src/pages/home/report/ReportActionItem.js

```diff
@@ -235,15 +236,48 @@ class ReportActionItem extends Component {
             <>
                 {children}
                 {hasReactions && (
-                    <ReportActionItemReactions
-                        reactions={reactions}
-                        toggleReaction={this.toggleReaction}
-                    />
+                    <View style={this.props.draftMessage ? styles.chatItemReactionsDraftRight : null}>
+                        <ReportActionItemReactions
+                            reactions={reactions}
+                            toggleReaction={this.toggleReaction}
+                        />
+                    </View>
                 )}
             </>
         );
     }
 
+    /**
+     * Get ReportActionItem with a proper wrapper
```

Comment:
What does a proper wrapper mean?

Reply:
Agreed, confusing a bit.

Would `Get ReportActionItem with draft item, single item or grouped items wrapper` make more sense?

Reply:
That sounds better.